### PR TITLE
Ability to request natural orientation (related to https://github.com…

### PR DESCRIPTION
**Pull request by vania-pooh**
…/appium/appium-android-bootstrap/pull/21)

### CODE REVIEW

File: lib/commands/actions.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ commands.getOrientation = async function () {
 
 commands.setOrientation = async function (orientation) {
   orientation = orientation.toUpperCase();
+  let params = {
+    orientation,
+    naturalOrientation: !!this.opts.androidNaturalOrientation,
+  };
   return await this.bootstrap.sendAction("orientation", {orientation});
```
Comment (imurchie) [inline lib/commands/actions.js]: This should be `return await this.bootstrap.sendAction("orientation", params);`

**Comment by vania-pooh** [comment]
Here Travis build seems to be broken.

File: lib/commands/actions.js (hunk 1 above)
Comment (imurchie) [inline lib/commands/actions.js]: This will also fix the build.

**Comment by vania-pooh** [comment]
@imurchie done.

**Comment by imurchie** [comment]
Cool. Let's see how Travis handles it, and then get this in!

**Comment by imurchie** [comment]
Sorry I didn't think of this earlier, but one more change needed. Can you add `androidNaturalOrientation` to https://github.com/appium/appium-android-driver/blob/master/lib/desired-caps.js#L123-L142 so that we don't get a message in the logs saying the desired capability is not recognized?

**Comment by vania-pooh** [comment]
@imurchie done.

**Comment by imurchie** [comment]
Published in `appium-android-driver@1.14.0`.

**Comment by vania-pooh** [comment]
@imurchie what is the estimate when these changes will be included in Appium release?

**Comment by imurchie** [comment]
If you re-install the current beta release (`npm uninstall -g appium && npm install -g appium@beta`) they will be there.

**Pull request merged**